src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/integration/TimeOutLicencesIntegrationTest.kt

```diff
@@ -61,28 +61,22 @@ class TimeOutLicencesIntegrationTest : IntegrationTestBase() {
     val jobExecutionDate = LocalDate.now()
     val isOnBankHolidayOrWeekEnd = workingDaysService.isNonWorkingDay(jobExecutionDate)
 
-    assertThat(timedOutLicences?.size).isEqualTo(if (isOnBankHolidayOrWeekEnd) 0 else 6)
-    assertThat(timedOutLicences)
-      .extracting<Tuple> {
-        tuple(it.licenceId, it.licenceStatus)
-      }
-      .contains(
-        if (isOnBankHolidayOrWeekEnd) {
-          tuple(1L, LicenceStatus.IN_PROGRESS)
-          tuple(2L, LicenceStatus.IN_PROGRESS)
-          tuple(4L, LicenceStatus.IN_PROGRESS)
-          tuple(7L, LicenceStatus.IN_PROGRESS)
-          tuple(8L, LicenceStatus.IN_PROGRESS)
-          tuple(9L, LicenceStatus.IN_PROGRESS)
-        } else {
-          tuple(1L, LicenceStatus.TIMED_OUT)
-          tuple(2L, LicenceStatus.TIMED_OUT)
-          tuple(4L, LicenceStatus.TIMED_OUT)
-          tuple(7L, LicenceStatus.TIMED_OUT)
-          tuple(8L, LicenceStatus.TIMED_OUT)
-          tuple(9L, LicenceStatus.TIMED_OUT)
-        },
-      )
+    if (!isOnBankHolidayOrWeekEnd) {
+      assertThat(timedOutLicences?.size).isEqualTo(4)
+
+      assertThat(timedOutLicences)
+        .extracting<Tuple> {
+          tuple(it.licenceId, it.licenceStatus)
+        }
+        .containsExactlyElementsOf(
+          listOf(
+            tuple(1L, LicenceStatus.TIMED_OUT),
+            tuple(2L, LicenceStatus.TIMED_OUT),
+            tuple(4L, LicenceStatus.TIMED_OUT),
+            tuple(9L, LicenceStatus.TIMED_OUT),
+          ),
```

Comment:
the other two expected items had ARD > CRD which we don't want to time out anymore